### DISCUSSION TITLE
Missing e-mail notification for unread report

### DISCUSSION THREAD

**Discussion by eleibr** [globaleaks/globaleaks-whistleblowing-software#3361] [General]
**Describe the bug**
The platform does not notify the Recipient with the reminder e-mail for unread reports 
 I assume it’s the unread_tips_mail_template you can find in the section Notification Settings -->Notification Templates
The Receiver associated with that context has not read the report and has not even logged into the platform, however he has only received the notification email of the expiring reports (72 hours before).

Version: 4.10.10

**To Reproduce**
-	Context --> set Expiration date (days): 5
-	Notification settings --> set Number of hours before sending a report expiration alert: 72
-	Advanced settings --> set Number of days till notifying unread reports to users: 2
-	File a report for the context 

**Expected behavior**
A notification e-mail should be sent to the relevant recipient as soon as the deadline expires.

**Comment by evilaliv3** [comment]
Thank you for reporting this @eleibr 

We will  try to reproduce the issue and get back to you.
